_posts/2015-05-04-aerogear-unified-push-alpha2-is-out.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: news
+section: news
+title: AeroGear UnifiedPush Server 1.1.0-alpha.2 released!
+author: matzew
+module: push
+releases: 1.1.0-alpha.2
+---
+
+
+We are happy to announce that the [Unified Push Server 1.1.0-alpha.2](https://github.com/aerogear/aerogear-unifiedpush-server/releases/tag/1.1.0-alpha.2) has been released ! 
```

Comment:
cghabged that

On Mon, May 4, 2015 at 12:43 PM, Christos Vasilakis <
notifications@github.com> wrote:

> In _posts/2015-05-04-aerogear-unified-push-alpha2-is-out.md
> <https://github.com/aerogear/aerogear.org/pull/582#discussion_r29576902>:
>
> > @@ -0,0 +1,19 @@
> > +---
> > +layout: news
> > +section: news
> > +title: AeroGear UnifiedPush Server 1.1.0-alpha.2 released!
> > +author: matzew
> > +module: push
> > +releases: 1.1.0-alpha.2
> > +---
> > +
> > +
> > +We are happy to announce that the [Unified Push Server 1.1.0-alpha.2](https://github.com/aerogear/aerogear-unifiedpush-server/releases/tag/1.1.0-alpha.2) has been released !
>
> the "description" title in the GH release notes page use 1.0.0-alpha.2.
> We should change to 1.1.0-alpha.2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/582/files#r29576902>.
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf